terraform/layers/base/s3.tf

```diff
@@ -0,0 +1,13 @@
+module "s3_bucket" {
+  source = "terraform-aws-modules/s3-bucket/aws"
+
+  bucket = "${var.project_name}-${var.environment}-webfiles"
```

Comment:
I saw on the previous PR you have created a local variable called 'resource_name_prefix', this is a good approach for standardising the naming on resources so I would reuse that here. Choosing either this or the prefix is fine, but we want to pick one approach and stick to the convention throughout the codebase to avoid confusion